sage-ios/SAGE/SAGE/CheckinRequestTableViewCell.swift

```diff
@@ -18,12 +18,14 @@ class CheckinRequestTableViewCell: UITableViewCell {
     var checkButton = UIButton()
     var xButton = UIButton()
     
+    var checkinID: Int?
```

Comment:
ooh but the button is the one sending the action, right? or does didSelectRowAtIndexPath get called too?